### PR TITLE
Python3 doctest compatibility: RetrieveRunInfo to SaveAscii-v2

### LINKED ISSUE #20820: Python3 doctest compatibility: RetrieveRunInfo to SaveAscii-v2
Python3 doctest compatibility: algorithms <RetrieveRunInfo  to SaveAscii-v2>

This issue is part of the move to Python3 in the doctests, 
Algorithms covered in this issue are:
```
RetrieveRunInfo-v1
RingProfile-v1
RotateInstrumentComponent-v1
RotateSource-v1
RunPythonScript-v1
SANSBeamFluxCorrection-v1
SANSConvertToWavelengthAndRebin-v1
SANSDarkRunBackgroundCorrection-v1
SANSFitShiftScale-v1
SANSStitch-v1
SANSWideAngleCorrection-v1
SCDCalibratePanels-v1
SassenaFFT-v1
SaveANSTOAscii-v1
SaveAscii-v2
```

### PR DESCRIPTION
**Pull request by brandonhewer**
Python3 doctest compatibility: algorithms <RetrieveRunInfo  to SaveAscii-v2>

## Description of work.
This PR is part of refactoring effort to bring Python3 compatiblity into the doctests.
See https://github.com/mantidproject/mantid/issues/20672 for details.

The following tests were refactored to be compatible :
```
RetrieveRunInfo-v1
RingProfile-v1
RotateInstrumentComponent-v1
RotateSource-v1
RunPythonScript-v1
SANSBeamFluxCorrection-v1
SANSConvertToWavelengthAndRebin-v1
SANSDarkRunBackgroundCorrection-v1
SANSFitShiftScale-v1
SANSStitch-v1
SANSWideAngleCorrection-v1
SCDCalibratePanels-v1
SassenaFFT-v1
SaveANSTOAscii-v1
SaveAscii-v2
```

Code review.

Fixes #20820 .
Internal change, no release notes

### CODE REVIEW

File: docs/source/algorithms/RotateInstrumentComponent-v1.rst [hunk 1]
```diff
@@ -169,16 +169,16 @@ Example 3: Rotating a single detector
...
   # Calculate the solid angles after rotation
   saws = SolidAngle( ws )
   sa2 = np.array( [ saws.readY(i)[0] for i in range(saws.getNumberHistograms()) ] )
 
   # Take element by element difference of the solid angles
   diff = sa2 - sa1
-  print diff
-  print 'The non-zero difference',diff[32] ,'is due to detector', ws.getDetector(32).getID()
+  print(diff)
+  print('The non-zero difference {} is due to detector {}'.format(diff[32], ws.getDetector(32).getID()))
```
Comment (rosswhitfield) [inline docs/source/algorithms/RotateInstrumentComponent-v1.rst]: I get different precision for python 3
```
Expected:
    The non-zero difference -0.0464531276188 is due to detector 33
Got:
    The non-zero difference -0.046453127618849614 is due to detector 33
```

File: docs/source/algorithms/SassenaFFT-v1.rst [hunk 2]
```diff
@@ -70,7 +70,7 @@ Usage
     ws = LoadSassena("loadSassenaExample.h5", TimeUnit=1.0)
     SassenaFFT(ws, FFTonlyRealPart=1, Temp=1000, DetailedBalance=1)
 
-    print('workspaces instantiated: ', ', '.join(ws.getNames()))
+    print('workspaces instantiated:  {}.format(', '.join(ws.getNames()))
```
Comment (rosswhitfield) [inline docs/source/algorithms/SassenaFFT-v1.rst]: You're missing a `'`

**Review by rosswhitfield — CHANGES_REQUESTED** [review]
There are also some `print` statements in `SANSBeamFluxCorrection, SCDCalibratePanels` that need to be fixed.

**Comment by josephframsay** [comment]
Brandon asked me to take over these PRs as he's not here until 16th

**Review by rosswhitfield — APPROVED** [review]

**Pull request merged**